### PR TITLE
Created functionality to automate Zanata upload

### PR DESCRIPTION
**Pull request by chalettu**
@miq-bot add_label housekeeping
@miq-bot add_label enhancement
@miq-bot add_label fine/no
@miq-bot add_label internationalization

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,3 +31,4 @@ addons:
     secure: khd4CM9YQns9dNWcPoYM7NizTalMXUyHtBtFTYoExD50Is/vpm99yc9QxicdUAKEAZZKZ1YVpoVBhuX0jzM+dX3XglLGpSmmX3vMu9JisimlZqOOK9EcTTTP+0cSkhvggCNYfB/V1kIloY2pektmS8C7V1RW79psTp1AGfnHwnrihGsfdTgjUJG18JO0EvEw0F/RobqWISU2A0UvRl03EndW4cWS1HCECsL/hamAwS94WzXI9+E2s9zA3E6V5+tLQnZdXOwzc5qNTh43e9GkaJ+kYFGvkTUWrWkoAZTGB/pHo8iCm7TbK1GpAz1WDXrkHMGfxIvezcxCxHWBR62c+wUJfTT6/v1RrYQ8Su9oQQ5fMB1B/evODevj31c5ctbdwOTbY/RMXYXlpzURGsrrf8TeqS77u+BhMeCUVjq0j1aSjc6dvzHJdL80luAs4lJjIdVDotGjQOjxek+St3zu9b3ZFLr0Dr2dH/UbCE4ABF0Ias0T0KUJjjxUM76SKV+SoRSIhsO046DSkjsQ8pWZIM+ywzAwHrPRdlvRHKmx/YgSTfr9v3LCOJnfac0e8IJdYftEzYOAMWcq+Pf5TWp8eBzstpvVNGE/bDQnXCWzypFlNi7+kmh7yu4alzqLPtBQTErCDe8tkINCH4wB8SnRHTFisVR/xidUT3YqPcocIZc=
 script:
 - yarn test
+- 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then yarn gettext:travis & yarn zanata:upload fi'
```
Comment (AllenBW) [inline .travis.yml]: Kinda curious, why create a new npm command rather than just adding `yarn gettext:extract & yarn gettext:compile` here?

**Comment by chalettu** [comment]
Status update on this PR.  We had an email exchange where concerns about sanitizing the catalog against HTML and Javascript being passed as strings into the catalog were a concern.  We are going to work on a regex sanitizer that will fail the build if the sanitization checks fail.

File: language/travis.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const shell = require('shelljs')
+
+if (process.env.TRAVIS_PULL_REQUEST && process.env.TRAVIS_PULL_REQUEST === 'false') {
+  const zanataUpload = shell.exec('yarn zanata:upload', { silent: true }).stdout
```
Comment (Fryguy) [inline language/travis.js]: What does the stdout look like?  I want to make sure nothing secret is exposed.

Comment (chalettu) [reply language/travis.js]: zanata
Here is what the std out looks like

**Comment by chalettu** [comment]
@chriskacerguis , @Fryguy has created the variables that will help this PR work properly after being merged.  We should be good to go.

**Comment by Fryguy** [comment]
@AllenBW @chriskacerguis This looks good to me, so merge when you are ready.

**Comment by Fryguy** [comment]
zanata version - manageiq-ui-service master 2017-10-02 17-54-11

🎉

**Comment by JPrause** [comment]
@chriskacerguis can you set the Milestone to Sprint 70

**Pull request merged**